cypress/integration/cf2-file-ui.js

```diff
@@ -0,0 +1,129 @@
+import {
+	visitPage,
+	getCfField,
+	clearCfField,
+	cfFieldIsVisible,
+	cfFieldDoesNotExist,
+	cfFieldHasValue,
+	cfFieldSelectValue,
+	cfFieldSetValue,
+	cfFieldCheckValue,
+	cfFieldIsDisabled,
+	cfFieldUnCheckValue,
+	cfFieldIsNotDisabled,
+	cfFieldCheckAllValues,
+	cfFieldCalcFieldValueIs
+} from '../support/util';
+
+
+describe('Name of test', () => {
+	beforeEach(() => {
+		visitPage('hello-world');
+	});
+
+	const formId = 'cf111';
+
+
+
+	it( 'Can add an image and see preview', () => {
```

Comment:
@New0 Use Jest and/or Enzyme to do a test with same spec directly on `FileInput`.

Reply:
Actually all of these tests should be possible using Jest and/ Or Enzyme

Reply:
@New0  BTW I added Enzyme and used it here: https://github.com/CalderaWP/Caldera-Forms/blob/feature/2766/clients/tests/unit/render/RenderComponentViaPortal.test.js